packages/babel-plugin-material-ui/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@material-ui/babel-plugin-material-ui",
+  "version": "1.0.0",
+  "description": "A simple transform to cherry-pick Material-UI imports",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "author": "Material-UI Team",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui-org/material-ui.git"
+  },
+  "bugs": {
+    "url": "https://github.com/mui-org/material-ui/issues"
+  },
+  "scripts": {
+    "build": "rollup -c",
```

Comment:
Do we need to involve rollup for building the plugin? Would using Babel be enough? I'm asking because it's what I use with babel-plugin-transform-react-remove-prop-types. It seems to be enough.

Reply:
No, don't need to involve babel either. We can use `tsc` to build the plugin, I only used rollup to get a single `index.js` file. But since that isn't important `tsc` will do just fine